src/main/java/org/opentripplanner/netex/mapping/RouteMapper.java

```diff
@@ -62,6 +74,8 @@
         org.opentripplanner.model.Route otpRoute = new org.opentripplanner.model.Route(
             idFactory.createId(line.getId())
         );
+
+        otpRoute.getGroupsOfLines().addAll(getGroupOfRoutes(line));
```

Comment:
```suggestion
        otpRoute.setGroupsOfLines(getGroupOfRoutes(line));
```
Note! If more than one set og GOL is added this will overwrite the existing set - that is ok if we know this is the only place this is done, if not we need to change the set method to an add method.